native/src/main/java/org/ballerinax/asb/util/ASBUtils.java

```diff
@@ -364,24 +358,27 @@ private static Object getValueWithIntendedTypeRecursive(byte[] value, Type type)
         return intendedValue;
     }
 
-    private static boolean hasStringType(UnionType type) {
-        return type.getMemberTypes().stream().anyMatch(memberType -> memberType.getTag() == STRING_TAG);
-    }
-
```

Comment:
@NipunaRanasinghe The SpotBugs validation detects this as an unused private function. Is there a reason to have it? If so, I can exclude it by adding a filter.

Reply:
+1 to remove